CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## 3.0.0-RELEASE
+
+### Changed
+* Using version 2 of the notification-api.
```

Comment:
should document the changes to `NotificationClient` constructor with the removal of service_id, api_key_id pattern